safe-room.lic

```diff
@@ -40,6 +40,9 @@ class SafeRoom
       hometown = town_data[settings.hometown]
       wait_at_empath(hometown['npc_empath']['id'])
       fix_standing
+    elsif settings.necro_safe_room_use_material
+      devour_healing(settings)
+      return
```

Comment:
Should this be here? This will skip over give_and_take below

Reply:
Removed it, thought give_and_take was something else.